[sfgui] Build error

**Host Environment**
- OS: Debian GNU/Linux bookworm/sid
- Compiler: g++ (Debian 11.2.0-19) 11.2.0

**To Reproduce**
Steps to reproduce the behavior:
`./vcpkg install sfgui`

**Failure logs**
vcpkg/buildtrees/sfgui/src/0.4.0-d0e06d1739.clean/src/SFGUI/ComboBox.cpp:213:25: error: ‘numeric_limits’ is not a member of ‘std’
  213 |         if( ( x == std::numeric_limits<int>::min() ) || ( y == std::numeric_limits<int>::min() ) ) {
      |                         ^~~~~~~~~~~~~~

**Additional context**
Including  &lt;limits&gt; header in ComboBox.cpp solvles the problem

--- a/ports/sfgui/002-add-limits-header.patch
+++ b/ports/sfgui/002-add-limits-header.patch
@@ -1,0 +1,13 @@
+diff --git a/src/SFGUI/ComboBox.cpp b/src/SFGUI/ComboBox.cpp
+index a01937b..454ec3e 100644
+--- a/src/SFGUI/ComboBox.cpp
++++ b/src/SFGUI/ComboBox.cpp
+@@ -9,6 +9,8 @@
+ #include <SFML/System/String.hpp>
+ #include <SFML/Window/Event.hpp>
+ 
++#include <limits>
++
+ namespace sfg {
+ 
+ // Signals.

--- a/ports/sfgui/portfile.cmake
+++ b/ports/sfgui/portfile.cmake
@@ -6,6 +6,7 @@ vcpkg_from_github(
     HEAD_REF master
     PATCHES
         "001-fix-corefoundation-link.patch"
+	"002-add-limits-header.patch"
 )
 
 file(REMOVE ${SOURCE_PATH}/cmake/Modules/FindSFML.cmake)

--- a/ports/sfgui/vcpkg.json
+++ b/ports/sfgui/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "sfgui",
-  "version-string": "0.4.0",
-  "port-version": 4,
+  "version": "0.4.0",
+  "port-version": 5,
   "description": "simple and fast graphical user interface library",
   "homepage": "https://github.com/TankOs/SFGUI",
   "dependencies": [

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -6430,7 +6430,7 @@
     },
     "sfgui": {
       "baseline": "0.4.0",
-      "port-version": 4
+      "port-version": 5
     },
     "sfml": {
       "baseline": "2.5.1",

--- a/versions/s-/sfgui.json
+++ b/versions/s-/sfgui.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "5a95a3a8728c4e2d8b71796f48492e699727d66b",
+      "version": "0.4.0",
+      "port-version": 5
+    },
+    {
       "git-tree": "64f6f7ee9963a096a2b4260c55be6228a22846e5",
       "version-string": "0.4.0",
       "port-version": 4

--- a/versions/s-/sfgui.json
+++ b/versions/s-/sfgui.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "5a95a3a8728c4e2d8b71796f48492e699727d66b",
+      "git-tree": "50f11f13413cb93d73d44bd0c00787363a00c98c",
       "version": "0.4.0",
       "port-version": 5
     },

